src/components/Console.jsx

```diff
@@ -3,6 +3,7 @@ import partial from 'lodash/partial';
 import React from 'react';
 import PropTypes from 'prop-types';
 import ImmutablePropTypes from 'react-immutable-proptypes';
+import prefixAll from 'inline-style-prefixer/static';
```

Comment:
Not quite sure what this does but I saw it in the other implementations

Reply:
@pwjablonski the `application.css` file is run though `autoprefixer`, which adds vendor prefixes for older browsers that don’t support the unprefixed version of more modern CSS features (e.g. `-webkit-flex` for `flex`). However, in the case we need to write styles directly into our JSX, `autoprefixer` doesn’t have a shot at the styles we’re writing. So we use `inline-style-prefixer` to do the same thing to inline styles.